python/paddle/fluid/tests/unittests/distributed_passes/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ if ((NOT WITH_GPU) AND (NOT WITH_XPU) AND NOT (WITH_ASCEND OR WITH_ASCEND_CL))
     list(REMOVE_ITEM TEST_OPS "test_auto_parallel_recompute_pass")
     list(REMOVE_ITEM TEST_OPS "test_auto_parallel_sharding_pass")
     list(REMOVE_ITEM TEST_OPS "test_auto_parallel_fp16_pass")
+    list(REMOVE_ITEM TEST_OPS "test_build_cinn_pass_resnet")
```

Comment:
The `test_build_cinn_pass_resnet.py` is skipped because it run too slow (about ~7min).

Reply:
Can we keep it? Since Paddle-CI-CINN CI time is not too long compared to Paddle-CI

Reply:
Done.